Dolphin scripts/Entrance Randomizer/lib/utils.py

```diff
@@ -44,6 +44,126 @@ def draw_text(text: str):
     _draw_text_index += 1
 
 
+def create_vertices(
```

Comment:
Let's move graph stuff to its own module. It's definitely gonna get expanded upon with item rando and whatnot.

Reply:
I agree, but that will be such a large commit that I would suggest we leave that for version 0.4.1 or something.

Reply:
It's not like I started reviewing that logic, and this PR is already big. Moving later would just create noise in the history and git blames. If we catch it early let's do the move now.

Reply:
Done.